src/pages/LoginPage/LoginFormConatiner/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,42 @@
+export default function LoginForm({ loginFields, onChange, onSubmit }) {
+  const { email, password } = loginFields;
+
+  const handleChange = (event) => {
+    const { target: { name, value } } = event;
+
+    onChange({ name, value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSubmit();
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label htmlFor="email">E-mail</label>
+        <input
+          id="email"
+          type="text"
+          name="email"
+          onChange={handleChange}
+          value={email}
+        />
```

Comment:
```suggestion
        <input
          type="text"
          id="email"
          name="email"
          value={email}
          onChange={handleChange}
        />
```
전 이렇게 읽기 편한순으로 작성해요 ㅎㅎ

Reply:
적어주신 것처럼 속성명 순서에 대한 가이드라인 같은 게 있을까요?

Reply:
가이드가 있다기보단 element 에서 중요하다고 생각되는 순서대로  작성하는 편이에요! input 이니까 type을 먼저 찾게 되더라고요 ㅎㅎ